nengo/builder/probe.py

```diff
@@ -33,10 +33,8 @@ def synapse_probe(model, key, probe):
         raise ValueError("Attribute '%s' is not probable on %s."
                          % (key, probe.obj))
 
-    if isinstance(probe.slice, slice):
+    if probe.slice is not None:
         sig = sig[probe.slice]
-    else:
```

Comment:
As far as I could tell it did, unless there is some weird indexing method I was missing.